specifications/ietf-eat-profile/spec.ocp

```diff
@@ -283,3 +283,119 @@ The following example illustrates a CWT containing claims for three target envir
+
+1. The EAT **MUST** be encoded as a signed CWT
+2. The EAT Profile claim (265) **MUST** be present and contain the OCP Profile OID
+3. The Nonce claim (10) **MUST** be present and contain the nonce value from the request
+4. The Measurements claim (273) **MUST** be present and contain concise evidence
+5. The issuer claim (1) **SHALL** be present if binding to a certificate chain
+6. The rim-locators claim (-70001) **MAY** be present to reference CoRIM locations
+
+## Transport Bindings
+
+### SPDM Binding
+
+When transported over SPDM, the GET_EAT command utilizes the VENDOR_DEFINED mechanism as follows:
+
+The GET_EAT request and response messages are transported as:
+- The Requester **MUST** use the SPDM VENDOR_DEFINED_REQUEST format
+- The Responder **MUST** use the SPDM VENDOR_DEFINED_RESPONSE format
+- The StandardID field **MUST** contain 4 (IANA)
+- The VendorID field **MUST** contain 42623 (OCP)
+- The first byte of the VendorDefinedReqPayload/VendorDefinedRespPayload is the Command Code, and must contain the value 02h to indicate GET_EAT
+- The GET_EAT request and response forms the payload in the VendorDefinedReqPayload and VendorDefinedRespPayload respectively, defined in the tables above
+
+For SPDM binding:
+- The SignerSlotID field **SHALL** correspond to SPDM certificate slot numbers
+
+### Native Transport Bindings
+
+TSM engines and other transport mechanisms **MAY** define their own bindings for the GET_EAT command, provided they:
+- Maintain semantic equivalence of request and response structures
+- Preserve all required fields and their meanings
+- Document any transport-specific adaptations
+
+## Security Considerations
+
+### Nonce Requirements
+- Requesters **SHOULD** use cryptographically secure random number generators for nonce generation
+- Nonces **SHOULD** be at least 32 bytes to prevent collision attacks
+- Requesters **MUST** verify that the response nonce matches the request nonce exactly
+
+### Signature Verification
+- Requesters **MUST** validate the COSE_Sign1 signature on the returned EAT
+- Certificate chain validation **MUST** be performed according to the requester's trust policy
+- The x5-chain in the unprotected header **MUST** be validated before trusting the EAT contents
+
+### Replay Protection
+- Requesters **SHOULD** maintain a cache of recently used nonces to detect replay attempts
+- Responses with duplicate nonces **MUST** be rejected
+- Nonce values **SHOULD NOT** be reused within a reasonable time window
+
+## Implementation Notes
+
+1. **Attesters** implementing GET_EAT:
+   - **MUST** support generating EATs for all provisioned attestation keys
+   - **SHOULD** complete GET_EAT requests within a reasonable timeout period
+   - **MAY** implement rate limiting to prevent denial of service
+
+2. **Requesters** using GET_EAT:
+   - **SHOULD** implement appropriate timeout handling
+   - **MUST** be prepared to handle ERROR responses
```

Comment:
SPDM ERROR responses?

Reply:
ack, might be good to clarify "error responses from the trasport". the capitalized ERROR sounds like it is referring to something specific.